android/sdk/src/main/java/org/jitsi/meet/sdk/OngoingConferenceTracker.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright @ 2019-present 8x8, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jitsi.meet.sdk;
+
+import android.app.Activity;
+
+import com.facebook.react.bridge.ReadableMap;
+
+import java.util.Collection;
+import java.util.Collections;
+import java.util.HashSet;
+
+class OngoingConferenceTracker {
```

Comment:
I would prefer if you would convert this utils class (lots of statics) to a singleton instance.
That way, it's easier to dispose of, do cleanup on it and/or provide a proxy/wrapper for it

Reply:
Ok.